features/scoped-custom-element-registries.yml

```
@@ -0,0 +1,11 @@
+name: Scoped custom element registries
+description: The `CustomElementRegistry()` constructor creates multiple custom element registries so that multiple custom element definitions for a single tag name can exist without conflicts. This is useful if more than one library is used, for example.
```

Strictly speaking the constructor doesn't create multiple registries on its own, only if you use it multiple times. What about this:

```suggestion
description: The `CustomElementRegistry()` constructor creates a new custom element registry that's separate from the global `window.customElements` registry. Creating more than one registry is useful to register multiple custom elements that have the same tag name.
```

Let me know if you think this is an improvement. This can probably be further improved.